BruteShark/PcapAnalyzerTest/NetworkMapModuleTest.cs

```diff
@@ -20,18 +21,24 @@ public NetworkMapModuleTest()
         public void NetworkMapAsJsonString_Test()
         {
             // Arrange
-            var connections = new List<PcapAnalyzer.NetworkConnection>();
-            var connection = new PcapAnalyzer.NetworkConnection(source: "1.1.1.1",destination: "2.2.2.2", protocol: "TCP", srcPort: 3009, dstPort: 80);
-
-            connections.Add(connection);
+            var connections = new List<PcapAnalyzer.NetworkConnection>() {
```

Comment:
@aviadtamir C# has a nice feature that called "Object Initializer", a way to initialize an object of a class or collection. Object initializers allow you to assign values to the fields or properties at the time of creating an object without invoking a constructor. I love it especially on data classes where no extra logic is needed at the constructor, less boilerplate code.

Reply:
👍 ok got you, now I understand what you meant on a prior comment when you said object initialzer